static/scripts/calendar.js

```diff
@@ -207,6 +227,12 @@ $(document).ready(() => {
 		viewRender(info) {
 			window.location.hash = info.view.name;
 		},
+
+		// viewSkeletonRender to trigger the addition of data-testid to navigation buttons
+		viewSkeletonRender() {
+			// Add custom data-testids for navigation buttons
+			addDatatestidsToCalendarNavigation();
+		},
```

Comment:
The comment lines are not necessary. The method name explains its aim itself :)

Reply:
Done